elgg-plugin.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+return [
+	'actions' => [
+		'account_removal/choices' => [],
+		'account_removal/confirm' => [],
+	],
+	'routes' => [
+		'collection:account_removal:confirm' => [
+			'path' => '/account_removal/confirm/{username}/{type}',
```

Comment:
this is different from the current url structure. Please update the usage where url's are regenated (Page menu and email notification)

Reply:
updated